lib/extensions/postgres_cdc_stream/cdc_stream.ex

```diff
@@ -43,21 +40,17 @@ defmodule Extensions.PostgresCdcStream do
     end
   end
 
-  @spec get_manager_conn(String.t()) :: {:error, nil} | {:ok, pid(), pid()}
+  @spec get_manager_conn(String.t()) :: nil | {:ok, pid(), pid()}
   def get_manager_conn(id) do
-    Phoenix.Tracker.get_by_key(Stream.Tracker, "postgres_cdc_stream", id)
-    |> case do
-      [] ->
-        {:error, nil}
-
-      [{_, %{manager_pid: pid, conn: conn}}] ->
-        {:ok, pid, conn}
+    case Phoenix.Tracker.get_by_key(Stream.Tracker, "postgres_cdc_rls_stream", id) do
+      [] -> nil
```

Comment:
I think you overwrote my last change from #682 here?

Reply:
reverting to what is in main